contents/handbook/company/structure.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Small Teams
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+PostHog is structured for speed, autonomy and innovation.
+
+Traditional organizations have big, separate functions. You have a product team, an engineering team, customer support, and so on. This slows things down, which stifles innovation - you have to get your boss to talk to someone else's boss to get work done. It also means that people can't really see the impact of their work.
+
+PostHog started off as a completely flat company with one big goal: to increase the number of successful products in the world.
+
+As we are getting bigger, we realized that it's harder for people to see the direct impact of their work, which reduces the sense of ownership.
+
+We have therefore introduced Small Teams. These are designed to each operate like a startup. 
+
+# How it works
+
+* A Small Team should never be more than ten people
+* A Small Team has an Accountable Person responsible for its performance - whoever is most appropriate depending on what the team is working on. Who would make the right founder for that bit of the company?
+* A Small Team must have (1) a customer, (2) a mission and (3) metrics
+* There may be certain functions where at our current stage we don't need a Small Team yet.
+* Each small team runs its own retrospective + sprint every week for 30 mins.
+
+# Small Teams list
+
+* Core analytics
+* Feature flags and a/b testing
+* VPC deployment
+* Website and docs
+* Extensibility: plugins and APIs
+* Session recording
+* Event capture (SDKs, autocapture)
+* Growth engineering (customer = _internal_ teams above)
```

Comment:
I think I'd probably suggest now that growth engineering *isn't* separate, but growth engineers could be join a Small Team with more need for this or could be the Accountable Person.

Reply:
I'd suggest growth is put under functional teams, where our customer is the business itself the product teams are collaborators with growth to accomplishing whatever the goal at the time is, depending on the business needs and resource allocations at the time.

Also I wonder if we should try applying one more level of grouping to these times – right now this list seems pretty feature based which I don't know if it'll be clear what a long term commitment is / area of ownership for a team is (as many of these features may only require a week or 2 of work at our current stage). 

Some example groupings...
'Core Experience' which encompasses core analytics + event capture
'Community' could encompass website, docs, slack
'Extensibility' could encompass plugins, APIs
'AB Testing' could encompass sessions, feature flags (though I might roll this into extensibility or core at the moment)

Reply:
Updated but iwll need a bit of refinement

Reply:
love the updates!